rules/order/src/Rector/Class_/OrderMethodsByVisibilityRector.php

```diff
@@ -90,23 +76,24 @@ private function privateFunctionName();
      */
     public function getNodeTypes(): array
     {
-        return [ClassLike::class];
+        return [Class_::class, Trait_::class];
     }
 
     /**
-     * @param ClassLike $node
+     * @param Class_|Trait_ $node
      */
     public function refactor(Node $node): ?Node
     {
-        if ($node instanceof Interface_) {
-            return null;
-        }
-
         $currentMethodsOrder = $this->stmtOrder->getStmtsOfTypeOrder($node, ClassMethod::class);
         $methodsInDesiredOrder = $this->getMethodsInDesiredOrder($node);
 
         $oldToNewKeys = $this->stmtOrder->createOldToNewKeys($methodsInDesiredOrder, $currentMethodsOrder);
 
+        // nothing to re-order
+        if (! $this->hasOrderChanged($oldToNewKeys)) {
+            return null;
+        }
```

Comment:
@dobryy Just few tips for next Rector rules.

This check prevents false positives on applied Rector rules:

![image](https://user-images.githubusercontent.com/924196/91657960-bab36f80-eac5-11ea-9f1c-650febda8372.png)


If node is returned, the rule says "I've changed this code". In this case, the order remained the same, so it was false positive. 

So `null` should be returned instead.

Reply:
Good to know, thank you!